react-components/select/step5.md

```diff
@@ -0,0 +1,34 @@
+Typeahead variants support adding new options, based on the user entered input. This is supported in two parts, first by allowing a user to select their custom input, and second by allowing this custom input to be added to the overall list of options for permanence.
+
+1. Add the isCreateable flag to the Select component.
```

Comment:
After the code block, say: this is what your select component should look like and add the code snippet